nodes/core/augmentations/compose_augmentation.py

```diff
@@ -76,8 +77,12 @@ def execute(
         if image is None and mask is not None:
             image = torch.zeros_like(mask)
 
-        image_tensor = TensorImage.from_BWHC(image) if isinstance(image, torch.Tensor) else None
-        mask_tensor = TensorImage.from_BWHC(mask) if isinstance(mask, torch.Tensor) else None
+        image_tensor = (
```

Comment:
Looks like your line length is set to 88? When I save this file with 120 characters line length it will be as the previous version.

Reply:
fixed

Reply:
I think it's still the case in some other files, e.g. `do_while_loop_end.py`.